Sources/StreamChat/Controllers/SearchControllers/UserSearchController/UserSearchController.swift

```diff
@@ -22,72 +22,29 @@ public class ChatUserSearchController: DataController, DelegateCallable, DataSto
     /// The `ChatClient` instance this controller belongs to.
     public let client: ChatClient
     
-    /// Filter hash this controller observes.
-    let explicitFilterHash = UUID().uuidString
-    
-    lazy var query: UserListQuery = {
-        // Filter is just a mock, explicit hash will override it
-        var query = UserListQuery(filter: .exists(.id), sort: [.init(key: .name, isAscending: true)])
-        // Setting `shouldBeObserved` to false prevents NewUserQueryUpdater to pick this query up
-        query.shouldBeUpdatedInBackground = false
-        // The initial DB fetch will return 0 users and this is expected
-        // In the future we'll implement DB search too
-        query.filter?.explicitHash = explicitFilterHash
-        
-        return query
-    }()
-    
     /// Copy of last search query made, used for getting next page.
-    var lastQuery: UserListQuery?
+    public private(set) var query: UserListQuery?
```

Comment:
Any reason why is this public now? And before it was internal? 🤔

Reply:
Moved it back to be internal